index.html

```diff
@@ -73,5 +73,168 @@ <h3 class="headline-connect">LET'S CONNECT</h3>
         </li>
       </ul>
     </section>
+    <!-- Work Section Container-->
+    <section class="work-section">
+      <!-- Work Section 1-->
+      <div class="work-info1">
+        <img src="/images/work-image1.png" alt="tonic" class="work-img" />
+        <h3 class="work-title">Tonic</h3>
+        <h4 class="profession">
+          CANOPY <span class="dot tag"> Back End dev </span>
+          <span class="dot tag">2015</span>
+        </h4>
+        <p class="title-description">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="stack-group">
+          <li class="stack">html</li>
+          <li class="stack">css</li>
+          <li class="stack">javascript</li>
+        </ul>
+        <button type="button" class="btn">See Project</button>
+      </div>
+
+      <!-- Work Section 1 -->
+
+      <div class="work-info2">
+        <img src="/images/work-image2.png" alt="tonic" class="work-img" />
+        <h3 class="work-title">
+          Multi-Post <br />
+          Stories
+        </h3>
+        <h4>
+          CANOPY <span class="dot tag"> Back End dev </span>
+          <span class="dot tag">2015</span>
+        </h4>
+        <p class="title-description">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="stack-group">
+          <li class="stack">html</li>
+          <li class="stack">css</li>
+          <li class="stack">javascript</li>
+        </ul>
+        <button type="button" class="btn">See Project</button>
+      </div>
+
+      <!-- Work Section -->
+
+      <div class="work-info3">
+        <img src="/images/work-image3.png" alt="tonic" class="work-img" />
+
+        <h3 class="work-title">Tonic</h3>
+        <h4>
+          CANOPY <span class="dot tag"> Back End dev </span>
+          <span class="dot tag">2015</span>
+        </h4>
+        <p class="title-description">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="stack-group">
+          <li class="stack">html</li>
+          <li class="stack">css</li>
+          <li class="stack">javascript</li>
+        </ul>
+        <button type="button" class="btn">See Project</button>
+      </div>
+      <!-- Work Section -->
+      <div class="work-info4">
+        <img src="/images/work-img4.png" alt="tonic" class="work-img" />
```

Comment:
The images on all your work sections are not displaying, you can fix the issue by fixing the link. Instead of ` <img src="/images/work-img4.png" alt="tonic" class="work-img" />`, replace it by ` <img src="./images/work-img4.png" alt="tonic" class="work-img" />`